pkgs/applications/editors/standardnotes/standardNotes.nix

```diff
@@ -0,0 +1,31 @@
+{stdenv, appimage-run}:
+
+stdenv.mkDerivation rec {
+  version = "2.3.11";
+  name = "standardNotes-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/standardnotes/desktop/releases/download/v${version}/standard-notes-${version}-x86_64.AppImage";
```

Comment:
If you are only going to support x86_64 you should specify this in meta.

Reply:
Just added support to i386, thanks for the review.